docs/getting-started/edfi-exchange/technology/the-north-allegheny-interventions-management-interface.md

```diff
@@ -0,0 +1,37 @@
+# The North Allegheny Interventions Management Interface
+
+<img src="https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/naTigerLogo.png" width="200" />
+
+# Description
+
+The Interventions Management Interface (IMI) - contributed by North Allegheny School District in Pittsburgh, PA - provides a lightweight student data reporting interface that doubles as an interventions and case management tool. The interface uses accessible platforms within the Microsoft environment, and is presented as a Power BI multi-page report with an embedded Power App module leveraging Microsoft’s Common Data Service for a simple model of non-ODS data structures.
+
+Within the interface, users can track student referrals for intervention, pre-referral and during-monitoring interventions, and case notes. For any student in the user district’s Ed-Fi ODS, users can drill through to review a student case profile, an attendance data overview (periodic and daily attendance), and a mini assessment results explorer. The tool also includes an ODS-wide assessment results explorer, allowing users to compare a selected student’s score against the full distribution of student scores on the same assessment.
+
+# Download
+
+* [Source Code](https://github.com/Ed-Fi-Exchange-OSS/Pupils-Personnel-Teams-Dashboard)
+
+## Guides
+
+* [Pre-requisites](https://github.com/Ed-Fi-Exchange-OSS/Pupils-Personnel-Teams-Dashboard/blob/main/prerrequisites.md)
+* [Installation and Configuration](https://github.com/Ed-Fi-Exchange-OSS/Pupils-Personnel-Teams-Dashboard/blob/main/Readme.md)
+* [User's Manual](https://github.com/Ed-Fi-Exchange-OSS/Pupils-Personnel-Teams-Dashboard/blob/main/usermanual.md)
+
+# Details
+
+* **By:** [North Allegheny School District](http://northallegheny.org)
+* **License Terms:** Ed-Fi License
+* **Released:** May, 2021
+
+# Contact
+
+* [Sean Hoover](mailto:shoover1@northallegheny.org)
+* [Richard Platts](mailto:rplatts@northallegheny.org)
+
+# **At a Glance**
+
+**Generation:** Tech Suite 3
+**For:** ODS / API v3.x
+
+![](https://edfi.atlassian.net/wiki/plugins/servlet/confluence/placeholder/unknown-macro?name=rate&locale=en_US&version=2)
```

Comment:
@roberthunterjr delete this line - remnants of a macro from before the TechDocs cloud migration